data/kramdown-rfc2629.erb

```diff
@@ -12,14 +12,28 @@
 <% end -%>
 ]>
 
-<% ps.arr("pi", false) do |pi, val| -%>
+<%
+  rfcattrs = ps.attrs("ipr", "docName=docname", "category=cat",
+                      "consensus", "submissionType=submissiontype", "xml:lang=lang",
+                      "number", "obsoletes", "updates", "seriesNo=seriesno")
+  if $options.v3
+    ps.arr("pi", false) do |pi, val|
+      keys = ["tocdepth", "toc", "sortrefs", "symrefs"]
+      if keys.index(pi) != nil && !ps.has(pi)
+        ps.add(pi, {true => "true", false => "false", nil => "true"}[val] || val)
```

Comment:
Why do we have to map nil to true here?

Reply:
That's what we do for the PIs.  I realize now that we also have to remap "yes" and "no" as well, as those were explicit values for PIs.

Reply:
Hmm, `tocDepth="true"`?

Reply:
Yes, it is possible to do non-correct things with this construction.

However, I've tested this structure and it works fine:

```yaml
pi:
  toc: true
  tocDepth: 100
```